packages/plexus/src/zoom/resetZoomIcon.tsx

```diff
@@ -18,10 +18,22 @@ import * as React from 'react';
 // https://github.com/react-icons/react-icons/blob/396b4d9241a61b6d22f3907273158b6a91aea2fd/md/zoom-out-map.js
 // LICENSE: MIT
 // https://github.com/react-icons/react-icons/blob/396b4d9241a61b6d22f3907273158b6a91aea2fd/package.json#L26
-export default (
+export const resetZoomIcon = (
   <svg fill="currentColor" preserveAspectRatio="xMidYMid meet" height="1em" width="1em" viewBox="0 0 40 40">
     <g>
       <path d="m35 25v10h-10l3.8-3.8-4.8-4.8 2.4-2.4 4.8 4.8z m-20 10h-10v-10l3.8 3.8 4.8-4.8 2.4 2.4-4.8 4.8z m-10-20v-10h10l-3.8 3.8 4.8 4.8-2.4 2.4-4.8-4.8z m20-10h10v10l-3.8-3.8-4.8 4.8-2.4-2.4 4.8-4.8z" />
     </g>
   </svg>
 );
+
+export const zoomInIcon = (
```

Comment:
Do we have to embed the icon? Everywhere else in the project we simply import from `react-icons`

Reply:
Actually in plexus the initial resetZoomIcon was embedded in the same way, I just followed the original convention and exported the svg in a similar manner.

Reply:
Can we switch to react-icons?

Reply:
I tried doing that, but it threw an error saying react icons should be listed in project dependencies, did not want to play with that so just followed the way the icons were already being used. Maybe there is some reason for creating an entire separate file only for the icons.

Reply:
So why not add them to package.json? We already have it in jaeger-ui:
```
packages/jaeger-ui/package.json:77:    "react-icons": "^4.10.1",
```

Reply:
Ok, will do that.

Reply:
Should I delete the ResetIcon.tsx file @yurishkuro?

Reply:
if it's not used / needed, sure